src/Kobo/Model.hpp

```diff
@@ -37,10 +37,11 @@ enum class KoboModel {
   GLO_HD,
   CLARA_HD,
   NIA,
+  LIBRA_H2O
 };
 
-gcc_const
-KoboModel
-DetectKoboModel();
+void InitKoboModel();
+
+KoboModel GetKoboModel();
```

Comment:
This is an API change which has nothing to do with Libra H2O, does it? This change obfuscates the commitdiff and makes review very hard. I don't even know why you're doing this; your commit message doesn't explain anything.